docker-compose.yml

```diff
@@ -39,8 +39,14 @@ services:
     # For development usage, use something like the following
     # build:
     #   context: https://github.com/Connexions/cnx-db.git#master
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+      - ${PWD}/20190805_dump.sql.gz:/docker-entrypoint-initdb.d/20190805_dump.sql.gz
```

Comment:
Not to throw a wrench in the gears, but this isn't necessarily the best way to go about doing this. I didn't mention it before, but I'm realizing I'd like a smaller container if possible.

It might be easier to do something like `gunzip 20190805_dump.sql.gz | docker-compose exec db psql -U rhaptos -d repository`. This way we aren't expanding the size the container takes up on disk.

A positive of doing this also means that one can start up the containers without loading the dump.

Reply:
Ah... yeah I guess we're always mounting the file this way...

Can probably do `docker-compose exec db /bin/bash -c 'psql $DB_URL'`